src/content/1.1/code/scala/snippet06.scala

```diff
@@ -0,0 +1,2 @@
+f compose identity[A] == f
+identity[B] _ compose f == f
```

Comment:
`_`  was a typo?

Reply:
@wi101 no, this is a way in Scala to convert a method to a function. This is called, I believe, eta-expansion, and there's a bit of explanation on how this works here https://scala-lang.org/files/archive/spec/2.11/06-expressions.html#method-values

Reply:
@hmemcpy thanks I didn't know about it!

Reply:
@hmemcpy ah yes! Sorry, I was confused it's :

identity[B](_) compose f

Reply:
@wi101 yeah, Scala is weird like that :)
All of the Scala code was checked with `tut`, so everything was compiling before I took the snippets from here: https://github.com/typelevel/CT_from_Programmers.scala